pkg/api/azenvtypes.go

```diff
@@ -80,6 +81,22 @@ var (
 		VnetCNILinuxPluginsDownloadURL:   "https://acs-mirror.azureedge.net/cni/azure-vnet-cni-linux-amd64-" + AzureCniPluginVerLinux + ".tgz",
 		VnetCNIWindowsPluginsDownloadURL: "https://acs-mirror.azureedge.net/cni/azure-vnet-cni-windows-amd64-" + AzureCniPluginVerWindows + ".zip",
 		ContainerdDownloadURLBase:        "https://storage.googleapis.com/cri-containerd-release/",
+		ImageRepoOverrides: map[string]ImageRepoOverride{
+			"k8s.gcr.io/hyperkube-amd64": {
+				Registry: "mcr.microsoft.com",
```

Comment:
See above comment about if this should be mcr or not.

Reply:
Where should it go to? Are all the versions available there?

Reply:
I have the following versions:

```
k8s/core/hyperkube-amd64:v1.12.6
k8s/core/hyperkube-amd64:v1.12.7
k8s/core/hyperkube-amd64:v1.13.5
k8s/core/hyperkube-amd64:v1.14.2
k8s/core/hyperkube-amd64:v1.15.0-beta.0
k8s/core/hyperkube-amd64:v1.16.0-alpha.0
```

Reply:
I tried building earlier versions (1.11 and 1.10) but something has changed with how those are built so my CI job cant build those versions.

Reply:
The full url would be `upstream.azurecr.io/k8s/core/hyperkube-amd64`

Reply:
Ok, I've updated. e2e tests will let us know if we are missing any required versions.